src/api/subscriptions.js

```diff
@@ -206,9 +206,10 @@ export let OffersRequest;
 export let LoginRequest;
 
 /**
- * @typedef {{
- *   token: string,
- * }}
+ * Properties:
+ * - one and only one of "token" or "authCode" and
+ * - corresponding value
```

Comment:
What does "corresponding value" mean?

Reply:
I mean to say the value corresponding to the key , token or authCode. Will remove if it is confusing.